docs/local-dev/cli-build-system.md

```diff
@@ -595,6 +595,33 @@ If you also want to enable source maps when debugging tests, you can do so by se
 }
 ```
 
+### Debugging Jest Tests
+
+To configure the debugging of your tests in IntelliJ, you have to:
+
+1. Enable in the root `tsconfig.json`
+
+```json
+{
+  ...
+  "compilerOptions": {
+    "sourceMap": true
+  }
+}
+```
+
+2. Create a Jest configuration by:
+
+- Click on "Edit Configurations" on top panel
+- Click "+"
+- And choose there "Jest"
+
+3. Fill in the next fields as:
+
+- Jest package: ~/proj/backstage/node_modules/jest
+- Working directory: (it depends on where is your test located, i.e. ~/proj/backstage/packages/core-plugin-api)
+- Jest config: --config ../../packages/cli/config/jest.js --runInBand
```

Comment:
Doesn't intellij have a way of dynamically saying "the root of the project"? Like a `%` or `$` variable or something? It's not great to have literally `~ /proj` in the docs, or that `../../` relative path which feels fragile.

Did you really have to point to the exact package in the second point? That's gonna be really annoying to have to change what package it points to every single time you want to run tests in a different package.

Reply:
Hi @freben thank you for your comments. 
Really valid points, let me describe another approach, hope you'll like it

Reply:
Updated the doc.

It's possible to update the Jest template once

<img width="1050" alt="Screenshot 2023-01-31 at 11 49 10" src="https://user-images.githubusercontent.com/2265094/215740121-b01d21fc-d1ec-41c0-ada3-671f9832f671.png">

IntelliJ doesn't like variables in that modal 🤷 

<img width="1633" alt="Screenshot 2023-01-31 at 11 52 13" src="https://user-images.githubusercontent.com/2265094/215740802-b7e156d1-8e4c-499e-b55a-8beca136fc4b.png">